extras/foreman_protector/dnf/foreman-protector.py

```diff
@@ -53,6 +55,8 @@ def _add_obsoletes(self):
         return final_query
 
     def sack(self):
+        if self.cli is not None and self.cli.command._basecmd not in PROTECT_COMMANDS:
```

Comment:
the `self.cli is not None` part is required so that e.g. Ansible (which uses the library, not the cli) still honors the excludes.

Reply:
isn't it due to safe access to `self.cli.command._basecmd` ? (to avoid `None. _basecmd` )

Reply:
kinda… dnf is smart enough to not choke on a failing plugin here, so instead of erroring out with an exception, the filtering just doesn't happen and ansible can install things w/o our supervision.